docs/examples/bitops+terraform+ansible-blog/README.md

```diff
@@ -5,12 +5,12 @@ Welcome to Bitops! This serves as a starting point for deploying your applicatio
 This repo can be run as is with
```

Comment:
First line in this README file:
> # Bitops Operations Repo
> Welcome to Bitops! This serves as a starting point for deploying your application to the cloud.


I'd recommend to include a minimal one-sentence description what it does under the hood,
similar to https://github.com/bitovi/bitops/pull/364/files#diff-c1d27a3e7a1e085fff9a6f46a1b1e3873e58c35211fcac08d666ff8777910e66R1